xbmc/windowing/wayland/WinSystemWaylandEGLContext.cpp

```diff
@@ -27,8 +27,11 @@ bool CWinSystemWaylandEGLContext::InitWindowSystemEGL(EGLint renderableType, EGL
 {
   VIDEOPLAYER::CRendererFactory::ClearRenderer();
   CDVDFactoryCodec::ClearHWAccels();
-
+#ifdef TARGET_WEBOS
```

Comment:
can you add a `using` declaration to CWinSystemWaylandEGLContext so we can get rid of all the #ifdefs in this file?

Reply:
I had to use typedef here, the compiler didn't accept using here

Reply:
there's no reason it shouldn't work and we don't allow `typedef`; please ping me on slack if you continue to have problems

Reply:
No idea what I was doing yesterday, was probably just too late 😅